test/offsetarray.jl

```diff
@@ -313,7 +313,7 @@ seek(io, 0)
 @test readdlm(io, eltype(A)) == parent(A)
 
 amin, amax = extrema(parent(A))
-@test clamp(A, (amax+amin)/2, amax) == clamp(parent(A), (amax+amin)/2, amax)
+@test clamp.(A, (amax+amin)/2, amax).parent == clamp.(parent(A), (amax+amin)/2, amax)
```

Comment:
the broadcast code will now check if this is zero preserving right? and return dense-in-sparse-format if not?